builds/e2e/connectivity.yaml

```diff
@@ -316,3 +318,4 @@ jobs:
           edgeRuntime.buildNumber: '$(edgeRuntime.buildNumber)'
           customEdgeAgent.image: '$(customEdgeAgent.image)'
           customEdgeHub.image: '$(customEdgeHub.image)'
+          runtimeLogLevel: '$(runtimeLogLevel)'
```

Comment:
For the consistency, please use `test.runtimeLogLevel` instead of `runtimeLogLevel`